packages/cli/src/controllers/tags.controller.ts

```diff
@@ -1,54 +1,72 @@
+import { UpdateTagRequestDto, CreateTagRequestDto } from '@n8n/api-types';
+import { RetrieveTagQueryDto } from '@n8n/api-types/src/dto/tag/retrieve-tag-query.dto';
```

Comment:
there is no `/src/` in released code. This will break.

Reply:
ahh wondering why the import added the source 🤔

Reply:
good catch